Warcraft.NET/Files/ADT/Entries/Legion/MBBBEntry.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.IO;
+using Warcraft.NET.Extensions;
+using Warcraft.NET.Files.Structures;
+
+namespace Warcraft.NET.Files.ADT.Entries.Legion
+{
+    /// <summary>
+    /// Blend mesh bounding box entry.
+    /// </summary>
+    public class MBBBEntry
+    {
+        /// <summary>
+        /// Unique ID of the map object.
+        /// </summary>
+        public uint MapObjectID { get; set; }
+
+        /// <summary>
+        /// Bounding box.
+        /// </summary>
+        public BoundingBox BoundingBox { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MBBBEntry"/> class.
+        /// </summary>
+        public MBBBEntry()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MBBBEntry"/> class.
+        /// </summary>
+        /// <param name="data">ExtendedData.</param>
+        public MBBBEntry(byte[] data)
+        {
+            using (var ms = new MemoryStream(data))
+            using (var br = new BinaryReader(ms))
+            {
+                MapObjectID = br.ReadUInt32();
+                BoundingBox = new BoundingBox(br.ReadVector3(AxisConfiguration.Native), br.ReadVector3(AxisConfiguration.Native));
```

Comment:
Maybe remove the Structures.AxisConfiguration.Native and use ReadBoundingBox/WriteBoundingBox